app/service/cloud_watch_service.rb

```diff
@@ -0,0 +1,28 @@
+class CloudWatchService
+  def self.week_submissions(form_id:)
+    region = "eu-west-2"
+    cloudwatch_client = Aws::CloudWatch::Client.new(region:)
+
+    response = cloudwatch_client.get_metric_statistics({
+      metric_name: "submitted",
+      namespace: metric_namespace.to_s,
+      dimensions: [
+        {
+          name: "form_id",
+          value: form_id.to_s,
+        },
+      ],
+      start_time: Time.zone.now.midnight - 7.days,
+      end_time: Time.zone.now.midnight,
+      period: 604_800,
```

Comment:
Can you set this to a variable/constant describing what the period means please? (it looks like a magic number - sorry if I missed this in forms-runner)

Reply:
You didn't miss it, this is just representing the previous week. I'll parse it out into something with proper names

Reply:
I've created some methods that better describe what points in time they're referring to